terratorch/datasets/generic_pixel_wise_dataset.py

```diff
@@ -154,18 +174,56 @@ def __getitem__(self, index: int) -> dict[str, Any]:
             "mask": self._load_file(self.segmentation_mask_files[index], nan_replace = self.no_label_replace).to_numpy()[0],
             "filename": self.image_files[index],
         }
+
         if self.reduce_zero_label:
             output["mask"] -= 1
         if self.transform:
             output = self.transform(**output)
         return output
-
+    
     def _load_file(self, path, nan_replace: int | float | None = None) -> xr.DataArray:
         data = rioxarray.open_rasterio(path, masked=True)
         if nan_replace is not None:
             data = data.fillna(nan_replace)
         return data
 
+    def _generate_bands_intervals(self, bands_intervals:List[List[int]] = None):
+        bands = list()
+        for b_interval in bands_intervals:
+            bands_sublist = np.arange(b_interval[0], b_interval[1] + 1).astype(int).tolist()
+            bands.append(bands_sublist)
+        return sorted(sum(bands, []))
```

Comment:
why sorted? Doesn't this destroy the order that the user requests?

e.g. If my `output_bands` are `[6-10]`, `[2, 4]`, doesn't this return `[2, 3, 4, 6, 7, 8, 9, 10]`? I would expect `[6, 7, 8, 9, 10, 2, 3, 4]`

Reply:
Also, no need for `np` and then `tolist()` here. Consider `list(range(b_interval[0], b_interval[1] + 1))`

Reply:
Really, will remove it.

Reply:
10551a0d59295101575540925bc6e49bc276e3fc